### ISSUE TITLE
NPE in ASkyBlock analysis

### ISSUE THREAD

**Issue by SlimeDog** [Rsl1122/Plan-PlayerAnalytics#375]
Spigot 1.12.2
Plan 4.0.2
ASkyBlock 3.0.8 (release a few minutes ago)

consol/log
```
[19:27:21 ERROR]: [Plan] A PluginData-source caused an exception: ${ASkyBlock_island_count}
[19:27:21 ERROR]: [Plan] main.java.com.djrapitops.plan.utilities.analysis.Analysis Caught java.lang.NullPointerException. It has been logged to Errors.txt
[19:27:21 ERROR]: [Plan] A PluginData-source caused an exception: ${ASkyBlock_island_level}
[19:27:21 ERROR]: [Plan] PluginData-source caused an exception: ${ASkyBlock_island_level} Caught java.lang.NullPointerException. It has been logged to Errors.txt
```
Errors.txt
```
PluginData-source caused an exception: ${ASkyBlock_island_level} Caught java.lang.NullPointerException
  com.djrapitops.pluginbridge.plan.askyblock.ASkyBlockIslandLevel.getValue(ASkyBlockIslandLevel.java:44)
  java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
  java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1548)
  java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
  java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
  java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
  java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
  java.util.stream.IntPipeline.collect(IntPipeline.java:472)
  java.util.stream.IntPipeline.average(IntPipeline.java:434)
  main.java.com.djrapitops.plan.utilities.analysis.MathUtils.averageInt(MathUtils.java:32)
  main.java.com.djrapitops.plan.utilities.analysis.AnalysisUtils.getAverage(AnalysisUtils.java:110)
  main.java.com.djrapitops.plan.utilities.analysis.Analysis.lambda$analyzeAdditionalPluginData$3(Analysis.java:196)
  java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
  java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
  java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374)
  java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
  java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291)
  java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
  java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
  java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
  java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
  java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

**Comment by Rsl1122** [comment]
I guess one of their API methods now return null.

**Comment by SlimeDog** [comment]
Could be. Code is supposed to check for and handle null.

**Comment by Rsl1122** [comment]
Yeah, it's my bad for not catching it, the code handling the PluginData calculations is pretty old.

**Comment by SlimeDog** [comment]
It clearly just changed in the 3.0.8 version released today. NHNF. :)

**Comment by Rsl1122** [comment]
Looked into the issue, seems that the cause of the NPE is not a value, but the API class is null instead.

**Comment by SlimeDog** [comment]
Yikes! Thanks for scoping it and opening the issue with @Tastybento.

**Comment by Rsl1122** [comment]
Fixed by tastybento in 3.0.8.1

**Comment by SlimeDog** [comment]
Good work! Thank you.
